.golangci.yml

```diff
@@ -32,5 +32,9 @@ linters-settings:
       - Standard
       - Default
       - Prefix(github.com/clusterpedia-io)
+  staticcheck:
+    checks:
+      - all
+      - "-SA1019" # disable the rule SA1019(Using a deprecated function, variable, constant or field)
```

Comment:
Is this necessarily related to this feature?
For version upgrades, I think it's important to ensure as few changes as possible, and this looks like it could create a new pr

Reply:
The reason has been added, thanks for the reminder